src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -962,7 +976,7 @@ void ImportCall(ILOpcode opcode, int token)
             {
                 for (int i = sig.Length - 1; i >= 0; i--)
                 {
-                    var actual = Pop();
+                    var actual = Pop(true);
```

Comment:
Nit: `Pop(allowUninitThis: true)` so it is immediately obvious what the argument is for.